Add abstract and bio for Ryan Whitehurst's talk

purplexa opened the pull request:
This adds the abstract for the talk by Ryan Whitehurst, titled
"Reproducible Infrastructure with Terraform and Puppet", along with an
accompanying speaker bio.

Commit:
Add abstract and bio for Ryan Whitehurst's talk

This adds the abstract for the talk by Ryan Whitehurst, titled
"Reproducible Infrastructure with Terraform and Puppet", along with an
accompanying speaker bio.

KrisBuytaert commented:
We have been discussing your talk proposal and wonder if you would be interested in giving it as an Ignite.
Ignites are 5 minute talks with 20 autoforwarding slides every 15 seconds.
They are the perfect vehicle to get your focussed message to a larger audience !

If you are interested, please let us know by updating this PR.

Commit:
Convert Ryan Whitehurst's talk to an Ignite

Per request, this reformats the abstract for Ryan Whitehurst's talk, titled
"Reproducible Infrastructure with Terraform and Puppet", into an Ignite instead
of a full talk.

Commit:
Adjust wording on Ryan Whitehurst's bio

purplexa commented:
@KrisBuytaert I think I could make it work as an Ignite. I updated the PR to add an Ignite entry instead of a presentation, as well as adjust the abstract a bit to account for the decreased scope and depth.
